classes/class-p4-campaign-exporter.php

```diff
@@ -60,18 +59,18 @@ public function single_post_export_bulk_action() {
 			$wp_list_table   = _get_list_table( 'WP_Posts_List_Table' );
 			$action          = $wp_list_table->current_action();
 			$allowed_actions = array( 'export' );
-			if ( ! in_array( $action, $allowed_actions ) ) {
+			if ( ! in_array( $action, $allowed_actions, true ) ) {
 				return false;
 			}
 			switch ( $action ) {
 				case 'export':
-					$sendback = 'admin.php?action=export_data&post=' . join( ',', $_REQUEST['post'] );
+					$sendback = 'admin.php?action=export_data&post=' . join( ',', array_map( 'absint', $_REQUEST['post'] ) );
 					break;
 
 				default:
 					return;
```

Comment:
Not part of the PR, but please make it return false here (like in line 63), so that inspection does not complain that there is no returning value.